src/static/js/modules/clear-form-buttons.js

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   Clear form button
+   - Clear checkboxes and selects
+   - Clear Chosen.js elements
+   - Clear jquery.custom-input elements
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  $( '.js-form_clear' ).on( 'click', function() {
+    var $this = $( this ),
+        $form = $this.parents( 'form' );
```

Comment:
Agreed. Lets make that a high priority next sprint so that it's clearer what's changing in future PRs